package.json

```
@@ -15,13 +15,13 @@
     "deploy:xdai": "graph deploy 1hive/honeyswap-xdai subgraph.xdai.yaml --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ --debug"
   },
   "devDependencies": {
-    "@graphprotocol/graph-cli": "^0.19.0",
-    "@graphprotocol/graph-ts": "^0.19.0",
+    "@graphprotocol/graph-cli": "0.37.2",
+    "@graphprotocol/graph-ts": "^0.29.1",
```

Could be worh it to get latest here with is "@graphprotocol/graph-ts@^0.29.3"
```suggestion
    "@graphprotocol/graph-ts": "^0.29.3",
```